site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -208,6 +267,84 @@ To remove a curve simply blank all boxes."></i>
         </div>
     </div>
 
+    <script>
+
+        function addToTable() {
```

Comment:
Would it be possible to write this function in a js file then call it in the twig file instead ?

Reply:
Thanks for pointing it out.